linux-cachyos-hardened/PKGBUILD

```diff
@@ -143,17 +141,37 @@ _build_nvidia_open=${_build_nvidia_open-}
 # Build a debug package with non-stripped vmlinux
 _build_debug=${_build_debug-}
 
-if [[ "$_use_llvm_lto" = "thin" || "$_use_llvm_lto" = "full" ]] && [ "$_use_lto_suffix" = "y"  ]; then
-     _pkgsuffix="cachyos-${_cpusched}-lto"
-elif [ "$_use_llvm_lto" = "none" ]  && [ -z "$_use_kcfi" ] && [ "$_use_gcc_suffix" = "y" ]; then
-    _pkgsuffix="cachyos-${_cpusched}-gcc"
+# Enable AUTOFDO_CLANG for the first compilation to create a kernel, which can be used for profiling
```

Comment:
AutoFDO isn't used by any other kernel except for the default and rc so there's not reason to add this.

Reply:
Whether its used or not is the decision of whoever is building the kernel, also, keeping it there keeps things consistent.

Reply:
If you want to add it in the name of consistency, then you should add it for the other PKGBUILDs too. The default kernel doesn't need to follow the other PKGBUILDs since it's already different in many ways (ThinLTO default, AutoFDO, more soon)